bokeh/models/tools.py

```diff
@@ -994,7 +994,7 @@ class HoverTool(Inspection):
     have a matching value for their ``name`` attribute will be used.
     """)
 
-    renderers = Either(Auto, List(Instance(Renderer)), defatult="auto", help="""
+    renderers = Either(Auto, List(Instance(Renderer)), default="auto", help="""
```

Comment:
Keyword arguments strike again. It's interesting that this wasn't picked up by any tests.

Reply:
Actually `"auto"` is the intrinsic default value and specifying `default` manually is just either pedantry or is supposed to aid documentation.